dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessDefinitionServiceImpl.java

```diff
@@ -1164,24 +1163,23 @@ private void importSubProcess(User loginUser, Project targetProject, ArrayNode j
     /**
      * get task node details based on process definition
      *
-     * @param defineIdList define id list
+     * @param defineCodeList define code list
      * @return task node list
      */
     @Override
-    public Map<String, Object> getTaskNodeListByDefinitionIdList(String defineIdList) {
+    public Map<String, Object> getTaskNodeListByDefinitionCodeList(String defineCodeList) {
         Map<String, Object> result = new HashMap<>();
 
         Map<Integer, List<TaskNode>> taskNodeMap = new HashMap<>();
-        String[] idList = defineIdList.split(",");
-        List<Integer> idIntList = new ArrayList<>();
-        for (String definitionId : idList) {
-            idIntList.add(Integer.parseInt(definitionId));
+        String[] codeArr = defineCodeList.split(",");
+        List<Long> codeList = new ArrayList<>();
+        for (String definitionCode : codeArr) {
```

Comment:
This can like:
Long[] codeArr = Arrays.stream(defineCodeList.split(Constants.COMMA)).map(Long::parseLong).toArray(Long[]::new);

Reply:
Here, build the Long list
I don't like the lambda style, haha

Reply:
OK